packages/variant-theme/src/ThemeGenerator.ts

```diff
@@ -0,0 +1,426 @@
+/* eslint-disable */
+/* tslint-disable */
+//@ts-nocheck
```

Comment:
```suggestion
```

Please don't do this - it completely defeats the purpose using typescript and eslint - if this was enabled some of my comments wouldn't be necessary because it would be caught by automation/tooling.

Reply:
fixed